src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -56,7 +61,15 @@ def connect_to_all_dbs_for_ns(namespace=DEFAULT_NAMESPACE):
         handle to all the dbs for a namespaces
     """
     db = swsscommon.SonicV2Connector(namespace=namespace)
-    for db_id in db.get_db_list():
+    db_list = list(db.get_db_list())
+    if not is_supervisor():
+        try:
+            db_list.remove('CHASSIS_APP_DB')
+            db_list.remove('CHASSIS_STATE_DB')
```

Comment:
Good to have an function returning back a list of chassis db instances in supervisor. 
But it is ok to do it later as well.